specs/interop/predeploys.md

```diff
@@ -133,7 +135,7 @@ function executeMessage(Identifier calldata _id, address _target, bytes calldata
 
     require(success);
 
-    emit ExecutingMessage(abi.encode(_id), _msg);
+    emit ExecutingMessage(keccak256(_msg), abi.encode(_id));
```

Comment:
Agree with @ajsutton above that we should just emit the `Identifier` struct directly instead of ABI encoding it to bytes first